### DISCUSSION TITLE
Do mods auto-update?

### DISCUSSION THREAD

**Discussion by harmlessgoose** [Albeoris/Memoria#717] [Q&A] [answered]
If I install a mod from the mod manager and a new version becomes available, does my existing version auto-update at a certain point, or do I have to manually update it somehow?

**Comment by snouz** [comment, marked as answer]
Hi, currently they do not auto-update, memoria doesn't check if a mod is outdated.

This is a feature we want to add though!

**Comment by harmlessgoose** [comment]
In that case, how do I know if a mod from the mod manager has been updated, and how do I update it myself?

**Reply by snouz** [reply]
There will be a version mismatch between the one on the catalog and the one in "Installed mods".
![image](https://github.com/user-attachments/assets/d1752393-bda9-4f5b-b065-10535fd70e8d)
To update, you can simply download it again, which will ask you if you want to replace your existing mod.

![image](https://github.com/user-attachments/assets/0ebf2e0c-b7bb-41e0-9929-4538f55c5a54)

**Reply by harmlessgoose** [reply]
Ah okay thanks! I think it would be great to improve this process, it currently seems very manual and a bit confusing, I'm a pretty tech savvy person but I couldn't quite figure that out. There could be an alert that mods are outdated with a button to update them, but the best outcome would be that mods autoupdate! Glad to know it's on your radar 🙂

Thanks for all the fantastic work you're doing on this to help me keep reliving a piece of my childhood! 🙏

**Reply by snouz** [reply]
It's a common request, and now that there's a big influx of people downloading mods from there, it should really be streamlined.

**Reply by harmlessgoose** [reply]
For example, if you make any incremental updates to Moguri 9, I want to make sure I'm getting those updates!
